src/webhint-theme/source/js/telemetry/telemetry.js

```diff
@@ -0,0 +1,114 @@
+"use strict";
+(function () {
```

Comment:
Should we log a task/bug to track sourcing this from the same code as @hint/utils-telemetry (used by the VS Code and browser extensions) once that's been updated to use the new endpoint? Probably also need a task to track migrating that code in the first place.